interfaces/seccomp/template.go

```diff
@@ -218,8 +218,8 @@ fadvise64
 fadvise64_64
 arm_fadvise64_64
 
-mbarrier
 mbind
+membarrier
```

Comment:
Do you think we should do a test that ensures all templates have sensible system call names. I had a patch that parsed this a while ago.